openedx/features/course_experience/utils.py

```diff
@@ -295,25 +291,3 @@ def reset_deadlines_banner_should_display(course_key, request):
                         display_reset_dates_banner = True
                         break
     return display_reset_dates_banner
-
-
-def can_show_verified_upgrade(user, course_id, enrollment):
```

Comment:
Is it possible to keep `can_show_verified_upgrade` in `openedx/`, and instead import `can_show_verified_upgrade` into `lms.courseware.utils`? Or would that cause an import cycle?

Cale and others confirmed that `lms/` should import from `openedx/`, not the other way around. I know edx-platform doesn't always follow that rule, but it would be good to keep new dependencies going in the "right" direction, if possible.

Reply:
@kdmccormick I understand according to your understanding of the dependency graph, LMS should depend on Openedx. However, that dependency relationship would not apply between `lms/djangoapps/courseware` and `openedx/features/course_experience`. If you look at the imports at https://github.com/edx/edx-platform/blob/master/openedx/features/course_experience/views/course_home.py#L17-L27, the `openedx/features/course_experience` depends on `lms/djangoapps/courseware`.

Reply:
@schenedx That is fair. I guess the ship has already sailed for `openedx/features/course_experience`.

I'll circle back with another review later today.